### PR TITLE
Updated external references (parameters.yml and responses.yml) in prep for next sprint's work

### PR DESCRIPTION
**Pull request by digitaljoshua**
Added a few parameters, added a few response codes

### CODE REVIEW

File: content/developer-guide/dev-guide-introduction.md [hunk 1]
```diff
@@ -3,4 +3,4 @@ uid: intro-dev-guide
 --- 
 # OSIsoft Cloud Services developer guide
 
-OSIsoft Cloud Services developer guide
\ No newline at end of file
+OSIsoft Cloud Services developer guide. This is what will appear for users when they click to the developer guide.
```
Comment (GiselleLeeOSI) [inline content/developer-guide/dev-guide-introduction.md:6]: Hi Josh, this content is no longer valid. See if you can delete this file from your PR. If unsure, talk to me.

File: content/external-references/parameters.yaml [hunk 2]
```diff
@@ -6,18 +6,46 @@ tenantId:
...
+    Client identifier
+id:
+  type: text/plain
+  value: |
+    Unordered list of Ids for all clients to get. Empty or whitespace Ids will be ignored.
+tag:
+  type: text/plain
+  value: |
+    Only return clients that have these tags
 query:
```
Comment (GiselleLeeOSI) [inline content/external-references/parameters.yaml:37]: query is tricky. There are two kinds: the one that is not supported and the other that is. And the unsupported one is already documented later called query-community. So you shouldn't change the description of this one here.

File: content/external-references/parameters.yaml [hunk 3]
```diff
@@ -6,18 +6,46 @@ tenantId:
...
   type: text/plain
   value: |
-    Query identifier
+    (Not supported) Query identifier
 count:
   type: text/plain
```
Comment (GiselleLeeOSI) [inline content/external-references/parameters.yaml:42]: Same for count and  and skip. You should keep the original definitions (these are actively being used). Instead you should create count and skip as they apply to identity and label them count-identity(or count-client) and skip-identity(skip-client) respectively.

**Review by GiselleLeeOSI — CHANGES_REQUESTED** [review]
1. Please take a look my comments before making changes. 
2. Please find way to uninclude dev-guide-introduction.md from this pr.

File: content/developer-guide/dev-guide-introduction.md (hunk 1 above)
Comment (digitaljoshua) [reply content/developer-guide/dev-guide-introduction.md]: i deleted this file from the PR (I think).

File: content/external-references/parameters.yaml (hunk 2 above)
Comment (digitaljoshua) [reply content/external-references/parameters.yaml:37]: reverted back to original definitions; created new query-identity, count-identity, skip-identity

File: content/external-references/parameters.yaml (hunk 3 above)
Comment (digitaljoshua) [reply content/external-references/parameters.yaml]: reverted back to original definitions; created new query-identity, count-identity, skip-identity

**Comment by digitaljoshua** [comment]
not sure if i made a mess of things. 

1. I removed the file content/developer-guide/dev-guide-introduction.md from this PR (hopefully I didn't remove it from the actual branch- I don't think I did but the messages for Git are a bit ambiguous)
2. Changed the modified definition of query back to its original verbiage. I'm sure there is a quick way of selecting that particular change in this PR and poof...cutting that out of the PR. But I just couldn't find it. So I copy/pasted the definition back to its original verbiage that way.
3. Ditto for definition of count
4. Ditto for definition of skip
5. Added 3 new parameters: query-identity, query-count, query-skip

File: content/external-references/parameters.yaml [hunk 4]
```diff
@@ -30,6 +70,34 @@ identityProviderId:
...
+  value: |
+    Number of identity providers to skip
+clientcredentialclientId:
+  type: text/plain
+  value: |
+    Client credential client identifier
+clientcredentialclientCount:
+  type: text/plain
+  value: |
+    Max number of roles to return
```
Comment (GiselleLeeOSI) [inline content/external-references/parameters.yaml:96]: Between 82 and 96, don't you want to be consistent in your use of Max vs maximum?

Comment (digitaljoshua) [reply content/external-references/parameters.yaml]: i fixed this

**Pull request merged**